doublets-ffi/examples/rust/examples/log-context.rs

```diff
@@ -0,0 +1,33 @@
+use doublets_ffi::{
+    export::{doublets_create_log_handle, doublets_free_log_handle},
+    FFICallbackContext,
+};
+use std::ffi::{c_char, CStr, CString};
+
+unsafe extern "C" fn callback(ctx: FFICallbackContext, ptr: *const c_char) {
+    let str = CStr::from_ptr(ptr).to_str().unwrap();
+    let ctx = &mut *(ctx as *mut usize);
+    *ctx += 1;
+
+    if *ctx % 2 == 0 {
+        print!("{str}");
+    } else {
+        eprint!("{str}");
+    }
```

Comment:
The difference between `print` and `eprint` is not noticeable. Use something more different

Reply:
More panic!

Reply:
Bro, panic abort my example :(